packages/calcite-components/src/demos/tip-manager.html

```diff
@@ -89,7 +89,7 @@
             <img
               slot="thumbnail"
               width="1000"
-              height="600"
+              height="200"
```

Comment:
Are these tweaks cosmetic? It's fine if they are, just for my understanding.

Reply:
Yeah just cosmetic. they looked rather tall. 😝